src/pygama/math/_distributions/double_gauss.py

```diff
@@ -0,0 +1,94 @@
+"""
+Double Gaussian distributions with different backgrounds for pygama
+"""
+import math
+
+import numpy as np
+
+from pygama.math._distributions.gauss import gauss_norm
+from pygama.math._distributions.step import step_pdf
+
+
+def Am_double(x,  n_sig1, mu1, sigma1,  n_sig2, mu2,sigma2, n_sig3, mu3,sigma3, n_bkg1, hstep1, n_bkg2, hstep2,
```

Comment:
I find these names for the Am utilities a bit cryptic, maybe `am241_double_peak_pdf`?

Reply:
rename it. something like double_gauss_step